frontend/src/lib/candidate/components/candidateContext/RequireLogin.svelte

```diff
@@ -2,9 +2,15 @@
   import {LoginPage} from '$lib/candidate/templates/login';
   import {getContext} from 'svelte';
   import type {CandidateContext} from '$lib/utils/candidateStore';
+  import type {Candidate} from '$lib/types/candidateAttributes';
+  import BasicPage from '$lib/templates/basicPage/BasicPage.svelte';
+  import Warning from '$lib/components/warning/Warning.svelte';
```

Comment:
Minor nitpick but this could match the style of other imports by using
`import {BasicPage} from '$lib/templates/basicPage'` and same with Warning